<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="myViewer">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="temperature">
         <attribute name="title">
          <string>Temperature</string>
         </attribute>
         <widget class="QLabel" name="temperature_text">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>0</y>
            <width>371</width>
            <height>121</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>48</pointsize>
           </font>
          </property>
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="pressure">
         <attribute name="title">
          <string>Pressure</string>
         </attribute>
         <widget class="QLabel" name="pressure_text">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>0</y>
            <width>371</width>
            <height>121</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>48</pointsize>
           </font>
          </property>
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="humidity">
         <attribute name="title">
          <string>Humidity</string>
         </attribute>
         <widget class="QLabel" name="humidity_text">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>0</y>
            <width>371</width>
            <height>121</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>48</pointsize>
           </font>
          </property>
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="read">
        <property name="text">
         <string>Read</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>429</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
